docs/getting_started/install.rst

```diff
@@ -186,6 +186,18 @@ To this::
     # Wrap the serve function with wagtail-cache
     url(serve_pattern, cache_page(wagtail_views.serve), name='wagtail_serve'),
 
+Caching specific wagtail pages only
```

Comment:
I would change the heading here to "Caching specific wagtail page *models* only" since it appears this code would cache all pages using a specific model, not specific page instances. Correct?

Reply:
Thanks for catching it. Updated PR.